hap.coffee

```diff
@@ -68,11 +68,13 @@ module.exports = (env) =>
         callback()
 
       @framework.on 'deviceAdded', (device) =>
-        accessory = @createAccessoryFromTemplate(device)
+        newAccessories = @createAccessoriesFromTemplate(device)
 
-        if accessory? && !accessory.exclude()
-          bridge.addBridgedAccessory(accessory)
-          env.logger.debug("successfully added device " + device.name)
+        if newAccessories?
+          for accessory in newAccessories
+            if accessory? && !accessory.exclude()
+              bridge.addBridgedAccessory(accessory)
+              env.logger.debug("successfully added device " + device.name)
```

Comment:
A last comment: would be good to use accessory.displayName instead of device.name, because for ButtonsDevices this would create the same log output for each button.